src/style.css

```
@@ -1 +1,77 @@
-/* Styles go here */
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  box-sizing: content-box;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  margin: 0;
+}
+
+header {
+  height: 60px;
+}
+
+.container {
+  max-width: 1200px;
+  padding: 10px 50px;
+  margin: auto;
+  display: flex;
+  justify-content: space-between;
+}
+
+.navigation {
+  display: flex;
+  align-items: center;
+}
+
+.navigation ul {
```

```suggestion
ul {
```
Дай класс этому элементу и не увеличивай специфичность